spec/lib/aggregators/test_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'spec_helper'
+
+describe Aggregators::Test do
+  it 'fetch can run as a future' do
+    future = Aggregators::Test.new.future.fetch
+    expect(future.value).to eq []
```

Comment:
Ik test niet of ik iets empties terug krijg, ik test of ik een lege array terug krijg. Even heel stom en heel specifiek.